image-layout.md

```diff
@@ -1,13 +1,13 @@
 ## Open Container Initiative Image Layout Specification
 
-The OCI Image Layout is a slash separated layout of OCI content-addressable blobs and [location-addressable](https://en.wikipedia.org/wiki/Content-addressable_storage#Content-addressed_vs._location-addressed) references (refs).
+The OCI Image Layout is a slash separated layout of OCI content-addressable blobs and [location-addressable](https://en.wikipedia.org/wiki/Content-addressable_storage#Content-addressed_vs._location-addressed) reference tags (tags).
```

Comment:
On Mon, Aug 29, 2016 at 3:16 PM W. Trevor King <notifications@github.com>
wrote:

> I'd rather keep both explicit: $ my-tool import LAYOUT-PATH REF [NAME]
> unless we decide to support something like HEAD [1], in which case REF
> would also be optional. If name was not specified on the command line, the
> name used for the imported object would come from some verifiable name
> (e.g. using the Merkle objects discussed in #176
> <https://github.com/opencontainers/image-spec/issues/176>, see [3]). [1]: #219
> (comment)
> <https://github.com/opencontainers/image-spec/pull/219#issuecomment-243197974>
> [2]: #219 (comment)
> <https://github.com/opencontainers/image-spec/pull/219#discussion_r76691546>
> [3]: #173 (comment)
> <https://github.com/opencontainers/image-spec/issues/173#issuecomment-234640781>
>

Eh, I think that it is going to be more common for someone to download "an
entire image named foo" then wanting to import the whole thing and expect
the tags to show back up rather than forcing the user to specify every ref.
For example you would actually need to do:

my-tool import /path/to/layout my-img-foo-bar-v1.0.0 my.img/foo/bar v1.0.0
my-tool run my.img/foo/bar:v1.0.0

vs

my-tool import /path/to/layout  my.img/foo/bar
my-tool run my.img/foo/bar:v1.0.0

I feel there is a usability cost to supporting lots of imgs in a single
layout on this common import UX.